include/rdma/fabric.h

```diff
@@ -318,6 +320,7 @@ struct fi_info {
 	fid_t			handle;
 	struct fi_tx_attr	*tx_attr;
 	struct fi_rx_attr	*rx_attr;
+	struct fi_rx_attr	*srx_attr;
```

Comment:
Shared rx attributes are already available from rx_attr when rx_ctx_cnt is set to FI_SHARED_CONTEXT.  See:

http://ofiwg.github.io/libfabric/v1.3.0/man/fi_endpoint.3.html